--- MLVPN-2.3.2.orig/doc/examples/mlvpn.init.d.in	2018-02-15 09:35:06.000000000 +0100
+++ MLVPN-2.3.2/doc/examples/mlvpn.init.d.in	2018-02-15 09:47:02.957193276 +0100
@@ -12,7 +12,7 @@
 # Default-Stop:      0 1 6
 # Short-Description: MLVPN Link Aggregator
 # Description: This script will start MLVPN tunnels as specified
-#              in @sysconfdir@/default/mlvpn and @sysconfdir@/mlvpn/*.conf
+#              in @sysconfdir@/sysconfig/mlvpn and @sysconfdir@/mlvpn/*.conf
 ### END INIT INFO
 
 # Original version by Robert Leslie
@@ -35,10 +35,10 @@
 
 [ -d $RUNDIR ] || mkdir $RUNDIR
 
-# Source defaults file; edit that file to configure this script.
+# Source sysconfig file; edit that file to configure this script.
 AUTOSTART="all"
-if test -e @sysconfdir@/default/mlvpn ; then
-  . @sysconfdir@/default/mlvpn
+if test -e @sysconfdir@/sysconfig/mlvpn ; then
+  . @sysconfdir@/sysconfig/mlvpn
 fi
 
 start_vpn () {
