diff -uarN doc/examples/systemd.orig/mlvpn-generator.in doc/examples/systemd/mlvpn-generator.in
--- doc/examples/systemd.orig/mlvpn-generator.in	2015-12-28 12:20:16.000000000 +0100
+++ doc/examples/systemd/mlvpn-generator.in	2018-01-23 11:09:05.930876061 +0100
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # This systemd generator creates dependency symlinks that make all MLVPN
-# tunnels listed in /etc/default/mlvpn's AUTOSTART be started/stopped/reloaded
+# tunnels listed in /etc/sysconfig/mlvpn's AUTOSTART be started/stopped/reloaded
 # when mlvpn.service is started/stopped/reloaded.
 
 set -eu
@@ -14,8 +14,8 @@
 
 mkdir -p "$WANTDIR"
 
-if test -e @sysconfdir@/default/mlvpn ; then
-    . @sysconfdir@/default/mlvpn
+if test -e @sysconfdir@/sysconfig/mlvpn ; then
+    . @sysconfdir@/sysconfig/mlvpn
 fi
 
 # No VPNs automatically started
diff -uarN doc/examples/systemd.orig/mlvpn.service.in doc/examples/systemd/mlvpn.service.in
--- doc/examples/systemd.orig/mlvpn.service.in	2015-12-28 12:20:16.000000000 +0100
+++ doc/examples/systemd/mlvpn.service.in	2018-01-23 11:11:17.499944150 +0100
@@ -11,6 +11,7 @@
 ExecStart=/bin/true
 ExecReload=/bin/true
 WorkingDirectory=@sysconfdir@/mlvpn
+EnvironmentFile=@sysconfdir@/sysconfig/mlvpn
 
 [Install]
 WantedBy=multi-user.target
diff -uarN doc/examples/systemd.orig/mlvpn@.service.in doc/examples/systemd/mlvpn@.service.in
--- doc/examples/systemd.orig/mlvpn@.service.in	2015-12-28 12:20:16.000000000 +0100
+++ doc/examples/systemd/mlvpn@.service.in	2018-01-23 11:11:44.530163530 +0100
@@ -6,10 +6,11 @@
 [Service]
 Type=notify
 NotifyAccess=main
-ExecStart=@sbindir@/mlvpn --config @sysconfdir@/mlvpn/%i.conf --user mlvpn
+ExecStart=@sbindir@/mlvpn --config @sysconfdir@/mlvpn/%i.conf --user mlvpn "$OPTIONS"
 ExecReload=/bin/kill -HUP $MAINPID
 WorkingDirectory=@sysconfdir@/mlvpn
 Restart=on-failure
+EnvironmentFile=@sysconfdir@/sysconfig/mlvpn
 
 [Install]
 WantedBy=multi-user.target
diff -uarN doc/examples/systemd.orig/mlvpn.sysconfig doc/examples/systemd/mlvpn.sysconfig
--- doc/examples/systemd.orig/mlvpn.sysconfig	1970-01-01 01:00:00.000000000 +0100
+++ doc/examples/systemd/mlvpn.sysconfig	2018-01-23 11:08:44.261700102 +0100
@@ -0,0 +1,8 @@
+#
+# Global start/restart/reload/stop options
+#
+OPTIONS=""
+
+# tunnels to be started automatically when mlvpn.service is started/stopped/reloaded.
+# Possible values: "all", "none", explicit list of tunnels
+AUTOSTART="all"
